sdk/python/kfp/cli/components.py

```diff
@@ -55,7 +55,7 @@
 {}/
 '''.format(_COMPONENT_METADATA_DIR)
 
-# Location at which v2 Python function-based components will stored
```

Comment:
It's okay to keep such in the comments, even though there's no v2 namespace any more, the concept of v2 version of component still exists.